bundle/Resources/views/ContentType/field_types.html.twig

```diff
@@ -116,6 +116,19 @@
     </div>
 {% endblock %}
 
+{% block ezobjectrelation_field_definition_edit %}
+    <div class="ezobjectrelation-settings selection-root">
+        {{- form_label(form.selectionRoot) -}}
+        {{- form_errors(form.selectionRoot) -}}
+        {{- form_widget(form.selectionRoot) -}}
+
+        <button data-universaldiscovery-title="{{"field_definition.ezobjectrelation.selection_root_udw_title"|trans({}, "ezrepoforms_content_type")}}"
```

Comment:
Please add `data-relation-selected-root-name-selector` attribute with selector to container which display content's name (ref: https://github.com/ezsystems/repository-forms/pull/31#issuecomment-117188410)